docs/rapid-pro.md

```diff
@@ -143,12 +145,12 @@ For details on modifying data in RAPID, see [Modifying Resources in RAPID](./rap
 
 ## Optional Features
 
-Although RAPID services can be very simple, because they follow core patterns they can be extended using optional Features,
-as appropriate, to support more advanced scenarios such as those described in [RAPID Features](./spec/rapid-pro-features.md).
+Although RAPID services can be very simple because they follow core patterns, they can be extended using optional Features
```

Comment:
```suggestion
Although RAPID services can be very simple because they follow core patterns, they can be extended using optional features
```

Lower-case?

Reply:
That makes sense, thank you!